packages/host/app/services/billing-service.ts

```diff
@@ -0,0 +1,11 @@
+import Service from '@ember/service';
+
+import ENV from '@cardstack/host/config/environment';
+const { stripePaymentLink } = ENV;
+
+export default class BillingService extends Service {
+  getStripePaymentLink(matrixUserId: string): string {
+    const clientReferenceId = encodeURIComponent(matrixUserId);
```

Comment:
From https://docs.stripe.com/payment-links/url-parameters:

>client_reference_id can be composed of alphanumeric characters, dashes, or underscores, and be any value up to 200 characters. Invalid values are silently dropped, but your payment page continues to work as expected.

```
encodeURIComponent("@jurgen:pepe.com")
'%40jurgen%3Apepe.com'
```

it looks like this will make non-alphanumeric values and stripe will drop the param